docs/_source/community/contributing.md

```diff
@@ -0,0 +1,81 @@
+
+Read our [Code of Conduct](https://github.com/argilla-io/argilla/blob/develop/CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read our [README](https://github.com/argilla-io/argilla/blob/develop/README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding good first issue](https://github.com/argilla-io/argilla/labels/good%20first%20issue)
+- [Find and review pull requests](https://github.com/argilla-io/argilla/pulls)
+- [Setting up the Argilla developer environment](./developer_docs.rst)
+- [Write a tutorial](https://github.com/argilla-io/argilla/issues/2030)
+- [Schedule a meeting with our developer advocate](https://calendly.com/argilla-office-hours/meeting-with-david-from-argilla-30m)
+
+## Getting started
+
+To navigate our codebase with confidence, see [our Python client code](https://docs.argilla.io/en/latest/reference/python/index.html) 🐍. For more information on how to use our UI, see [our web app reference](https://docs.argilla.io/en/latest/reference/webapp/index.html) 💻.
+
+We accept three kinds of contributions.
+- [documentation](#features-and-documentation): better textual support for features. Best to get started.
+- [issues](#issues): bug reports that directly fix something that is broken. Generally smaller in nature.
+- [features](#features-and-documentation): new functionalities. Generally larger in nature.
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs or our code, [search if an issue already exists](https://github.com/argilla-io/argilla/issues?q=is%3Aissue). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/argilla-io/argilla/issues/new?assignees=&labels=bug&template=bug_report.md&title=). Try to fill out the entire template to ensure it is clear what the issue entails and it is reproducible for other users.
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/argilla-io/argilla/issues?q=is%3Aissue) to find one that interests you. You can narrow down the [search using `labels`](https://github.com/argilla-io/argilla/labels) as filters. If you find an issue to work on, you are welcome to [create a fork and start making changes](#make-changes). However, we always advise you to ask for some input within the discussion on the issue, or by contacting us via Slack or by [scheduling a meeting](https://calendly.com/argilla-office-hours/meeting-with-david-from-argilla-30m).
+
+### Features and documentation
+
+We are always looking for [new tutorials](https://github.com/argilla-io/argilla/issues/2030)!
```

Comment:
```suggestion
#### Contribution to the docs
You can contribute to the docs by
- Writing a tutorial about a specific topic. We are always looking for [new tutorials](https://github.com/argilla-io/argilla/issues/2030) and we would love to support you!
- Including missing parts, examples, improving the writing, fixing typos, etc.

For contributing to the docs, here are two references:
1. How to locally build the docs
2. A template for tutorials

```

Reply:
Include the links to the dev guide where we say how to build the docs and the tutorial template included in this PR